k8s/openebs-operator.yaml

```diff
@@ -37,7 +37,13 @@ rules:
   verbs: [ "get", "list", "create" ]
 - apiGroups: ["*"]
   resources: ["storagepools"]
-  verbs: ["get", "list"] 
+  verbs: ["get", "list"]
+- apiGroups: ["volumesnapshot.external-storage.k8s.io"]
+  resources: ["volumesnapshots"]
+  verbs: ["get", "list", "watch", "create", "update", "patch", "delete"]
+- apiGroups: ["volumesnapshot.external-storage.k8s.io"]
+  resources: ["volumesnapshotdatas"]
```

Comment:
similar to pvc, pv. we can put  volumesnapshotdatas and volumesnapshots into same resources list.

Reply:
Done.

Reply:
Yes, Done !